metadata.tex

```diff
@@ -278,7 +289,13 @@
 \maketitle
 
 \begin{center}
-  \version{}
+  \getenv[\githubsha]{GITHUB_SHA}
```

Comment:
Gosh, I had no idea LaTeX had such capabilities. I'm a little uneasy with adding an extra package just for this, but on the other hand I like the parametrization, which simplifies the workflow.

Reply:
How about putting this under another iftoggle, which kicks in only on GitHub? It should allow us to bypass the TeXstudio error?

Reply:
That could work. How to detect the github env?

Reply:
Just define another `iftoggle`, and pass it over in the `pre_compile:` arg of the LaTeX GitHub action.

Reply:
Something like `githubbild`?

Reply:
Exactly the same way that `printable` is passed, but make use it on GitHub only.